dev/breeze/src/airflow_breeze/commands/release_command.py

```diff
@@ -49,25 +50,13 @@ def create_version_dir(version):
         console_print(f"{version} directory created")
 
 
-def copy_artifacts_to_svn(rc, svn_dev_repo):
+def copy_artifacts_to_svn(rc, svn_dev_repo, svn_version_dir):
     if confirm_action(f"Copy artifacts to SVN for {rc}?"):
-        run_command(
-            [
-                "for",
-                "f",
-                "in",
-                f"{svn_dev_repo}/{rc}/*",
-                ";",
-                "do",
-                "svn",
-                "cp",
-                "$f",
-                "${$(basename $f)/}",
-                "done",
-            ],
-            dry_run_override=DRY_RUN,
-            check=True,
-        )
+        dev_rc_path = f"{svn_dev_repo}/{rc}/"
+        if not DRY_RUN:
+            for filename in os.listdir(dev_rc_path):
+                source_file = os.path.join(dev_rc_path, filename)
+                shutil.copy(source_file, svn_version_dir)
```

Comment:
I think we should continue to use svn cp. Most notably, this will use more disk space I believe.

Reply:
It also keep history in the Foundation repo. The copy is recorded in SVN and allows to track provenience of files.

Reply:
How do we correct the command because it didn't work when I used it? The error was "no such file 'for'". Maybe peculiar to M1?